common.mk

```diff
@@ -236,6 +225,24 @@ dramsim_lib = $(dramsim_dir)/libdramsim.a
 $(dramsim_lib):
 	$(MAKE) -C $(dramsim_dir) $(notdir $@)
 
+################################################
+# Helper to run SBT or manage the SBT server
+################################################
+
+SBT_COMMAND ?= shell
+.PHONY: launch-sbt
+launch-sbt:
+	cd $(base_dir) && $(SBT_NON_THIN) "$(SBT_COMMAND)"
+
+.PHONY: shutdown-sbt-server
+shutdown-sbt-server:
```

Comment:
We should probably use the same commands across both firesim / CY.

Reply:
Sure.